setup.cfg

```diff
@@ -109,7 +110,7 @@ testpaths = "sunpy" "docs"
 norecursedirs = ".tox" "build" "docs[\/]_build" "docs[\/]generated" "*.egg-info" "examples" "sunpy[/\]_dev" ".jupyter" ".history" "tools" "sunpy[\/]extern" ".pyinstaller"
 doctest_plus = enabled
 doctest_optionflags = NORMALIZE_WHITESPACE FLOAT_CMP ELLIPSIS
-addopts = --doctest-rst --doctest-ignore-import-errors -p no:unraisableexception -p no:threadexception
+addopts = --dist no --doctest-rst --doctest-ignore-import-errors -p no:unraisableexception -p no:threadexception
```

Comment:
Why did you add `--dist no` here? What's it achieving that isn't by default? Don't you have to specify `-n` to enable pytest-xdist?

Reply:
Why do we want to ensure pytest-xdist is installed? To be clear I don't care that it's here, I just want to understand why.

Reply:
Not a clue. At least this way, it ensures its installed and listed as a test deps instead of only working via tox/CI.